.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -0,0 +1,32 @@
+name: "publish-technical-documentation-next"
+
+on:
+  push:
+    branches:
+    - "main"
+    paths:
+    - "docs/sources/**"
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+    - name: "Check out code"
+      uses: "actions/checkout@v3"
+    - name: "Build website"
+      run: |
+        docker run -v ${PWD}/docs/sources:/hugo/content/docs/mimir/latest -e HUGO_REFLINKSERRORLEVEL=ERROR --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
+
+  sync:
+    runs-on: "ubuntu-latest"
+    needs: "test"
+    steps:
+    - name: "Publish to website repository (next)"
+      uses: "./.github/actions/website-sync"
```

Comment:
Isn't missing this action? I think we need to download it first.

Reply:
Added in [f606d3c](https://github.com/grafana/mimir/pull/1820/commits/f606d3cfeaeea8beb01960854a8810dce1d9a114)